samples/tab-external-auth/csharp/README.md

```diff
@@ -0,0 +1,115 @@
+---
+page_type: sample
+description: Microsoft Teams sample app which showcases google authentication using external auth providers.
```

Comment:
Add proper manifest images, you have added some new images with different width.

Reply:
changed